src/app/main/time-groups/edit-time-group/edit-time-group.component.ts

```diff
@@ -188,21 +191,35 @@ export class EditTimeGroupComponent implements OnInit {
     }
   }
   public nextpage() {
-    let nextPage = this.employeesPaginationData.next_page;
-    this.employessParams.page = nextPage;
-    if (nextPage) {
-      this.getEmployees()
+    if (this.employeesPaginationData.next_page) {
```

Comment:
I think nextPage should be disabled if there is no next page in the pagination params so we don't need to handle it here